### PR TITLE
ADDITION OF SECTION AND ABOUT PAGE ON MY MOBILE PORTFOLIO

### PR DESCRIPTION
**Pull request by citec-47**
#ADDITION OF SECTION AND ABOUT PAGE ✔✔✔
# SET-UP✔✔
#General requirements🧾
-Make sure that there are [no linter errors] (https://github.com/microverseinc/linters-config):
-Linter should be set up correctly for this repository.
-Linter result should be green.
-There should be no custom changes to linter config.
*Make sure that you used correct [GitHub Flow] (https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git--github/articles/github_flow.md):
-README file should be descriptive.
-Commit messages should be meaningful.
*Pull Request should have a descriptive title telling you what changes are inside.
-Pull Request should have a short summary describing introduced changes in more detail.
-Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal skills/blob/main/documentation/articles/professional_repo_rules.md):
-Branch should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
-Pull Request is made from 'feature branch' into the main branch.
HTML/CSS requirements🧾🌳
-Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
-Project requirements
-IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. It is the first time you need to work with multiple pull requests in your one repository. [Read how to do it most efficiently](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

🧾Now you will build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:

#"Works" section
-Toolbar

-the "About myself" section (placed after the projects and before the contact form):
-About myself

-Make sure the content of the "About myself" section is aligned with flexbox.

-In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

-Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items.

-Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

-You don't need to implement the detail modal window that requires JavaScript.

-You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    <ul>
+        <li><a href="https://twitter.com/home"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="https://www.linkedin.com/feed/?trk=homepage-basic_google-one-tap-submit"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+        <li><a href="https://github.com/citec-47"><i class="fa-brands fa-github"></i></a></li>
+        <li><a href="https://www.facebook.com/ndonyi.maurice"></i></a></li>
+     </ul>
+    </section>
+</section>
+<section>
```
Comment (ABDULALI3468) [inline index.html:46]: Please consider using `display: grid;` for the container to align the elements as its a project requirement.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
+        </div>
+    </div>
+   
+  </div>
```
Comment (ABDULALI3468) [inline index.html]: Currently, your styles are not exactly lookalike to the Figme design template you have chosen. Remember your style needs to be like a template at least above 375px until it hits desktop size. Currently, they are not as it's flowing out of their container height. I am attaching a screenshot of your design to make it more clear. 

## Your Design:-

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="all">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
```
Comment (ABDULALI3468) [inline index.html]: Please consider adding the functionality of press on the button. So when the button is pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file.

.resume:active{
    background-color: #2230D2;
    color: white;
}

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+      <h3 class="lets">LET’S CONNECT</h3>
+      <ul class="list_holder">
+          <li class="list"><a href="https://twitter.com/home"><i class="fa-brands fa-twitter"></i></a></li>
+          <li class="list"><a href="https://www.linkedin.com/feed/?trk=homepage-basic_google-one-tap-submit"><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li class="list"><a href="#"><i class="fa-solid fa-m"></i></a></li>
+          <li class="list"><a href="https://github.com/citec-47"><i class="fa-brands fa-github"></i></a></li>
+          <li class="list"><a href="https://www.facebook.com/ndonyi.maurice"><i class="fa-brands fa-facebook"></i></a></li>
+       </ul>
+      </section>
+      <button class="resume">Get my resume</button>
```
Comment (ABDULALI3468) [inline index.html]: Please consider adding the functionality of press on the button. So when the button is pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file.

.resume:active{
    background-color: #2230D2;
    color: white;
}

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+      <h1>Contact me</h1>
+      <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
+    </div>
+    <form action="post">
+      <input type="text" value="Ndonyi" placeholder="Your Name Here" id="input" required/>
+      <input type="email" value="mauricendonyi40@gmail.com" placeholder="Your Email Here" id="input" required/>
+      <textarea id="textpanel" cols="30" rows="9" placeholder="Write Your Message Here"></textarea> 
+      <submit type="submit" class="submit">Get In Touch</a></submit>
+    </form>
+  </section>
```
Comment (ABDULALI3468) [inline index.html]: Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

Following is the code representation of the above if you are confused about what the instructions mean:-


`    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Python</li>
          <li>Java</li>
        </ul>
      </li>
      <li>Frameworks</li>
      <li>Skills</li>
    </ul>`


image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+          <li>JAVASCRIPT</li>
+        </ul>
+        <br>
+        <br>
+        <a href="https://github.com/citec-47/PORTFOLIO-TWO/blob/set-up/README.md" class="resume">See Project</a>
+      </div>
+  </div>
+ </div>
+ </section>
+<section class="about_me">
```
Comment (ABDULALI3468) [inline index.html:157]: Please consider reconsidering your design to match the Figma template. It includes similar margins in this case etc. Moreover, on the right side of Languages, there needs to be an icon that is not showing consider rechecking it.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @citec-47 , 



 Indeed you did an exceptional job but still, there are a few things that need to be addressed to go to the next level.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by citec-47** [comment]
#CHANGES DONE🌳
-Addition of the hover effects✔
-adding display grid on the contact-list section✔
-using ul instead of div on the about section✔
-making sure the minimum width should be 375px as prescribe for the exercise✔
image

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @citec-47 :clap: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 

## Optional suggestions

_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**